### PR TITLE
chore(github): update feature request issue template

### PR DESCRIPTION
**Pull request by rderbier**
## Problem
The current template is more a bug report and does not ask for references, current workaround and suggested solution.

## Solution
update the template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 1]
```diff
@@ -6,44 +6,52 @@ body:
   - type: markdown
     attributes:
       value: |
-        Thank you for reporting an issue.
-        If you think we can make Dgraph better, describe your feature request here.
-        Note: Feature requests are judged based on user experience and modeled on 
-        [Go Experience Reports](https://github.com/golang/go/wiki/ExperienceReports). 
-        These reports should focus on the problems - they should not focus on and need not propose solutions.
-  - type: dropdown
+      You have an idea for making Ratel better? We'd love to hear about it.
```
Comment (MichelDiz) [inline .github/ISSUE_TEMPLATE/feature_request.yml:9]: Why Ratel? This repo is for Dgraph Core.

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -6,44 +6,52 @@ body:
...
+      label: Use case
+      description: Describe the context in which the feature will be used and what is achieved when the feature is used.
+      placeholder: |
+          Type here.
     validations:
-      required: false
+      required: true
   - type: textarea
     attributes:
-      label: What you wanted to do.
```
Comment (MichelDiz) [inline .github/ISSUE_TEMPLATE/feature_request.yml:24]: I'm not sure about removing these step questions.

What you wanted to do.
What you actually did.
Why wasn't it great, with examples.

have you discuss with someone about this?

Comment (rderbier) [reply .github/ISSUE_TEMPLATE/feature_request.yml:24]: I checked feature request templates on other github repos ( I found nice guides in RStudio repo). Feature request are usually for something new or not possible. So the current questions does not seem to make much sense in that context IMO (more for Bug report). 
But I also created the PR to get engineering feedback on what they need in a Feature Request.

**Review by MichelDiz — APPROVED** [review]

**Review by matthewmcneely — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/enhancement.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
+name: "⚡ Enhancement Request"
+description: Something could be better.
+title: "[ENHANCEMENT]: <Title>"
+labels: ["kind/enhancement "]
```
Comment (skrdgraph) [inline .github/ISSUE_TEMPLATE/enhancement.yml:4]: extra space at end - change to `"kind/enhancement"`

File: .github/ISSUE_TEMPLATE/enhancement.yml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Something could be better?
+        If you request is about a net new feature please use the Feature Request template.
```
Comment (skrdgraph) [inline .github/ISSUE_TEMPLATE/enhancement.yml:10]: type `If you` -> `If your`

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 5]
```diff
@@ -1,49 +1,56 @@
 name: "⚡ Feature Request"
 description: Suggest a new feature
 title: "[FEATURE]: <Title>"
-labels: ["kind/feature ", "status/triage"]
+labels: ["kind/feature"]
```
Comment (skrdgraph) [inline .github/ISSUE_TEMPLATE/feature_request.yml:4]: the nice part of keeping `"status/triage"` is that we know what we have NOT triaged.

File: .github/ISSUE_TEMPLATE/enhancement.yml (hunk 3 above)
Comment (skrdgraph) [reply .github/ISSUE_TEMPLATE/enhancement.yml:4]: also, add `"status/triage"`

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 6]
```diff
@@ -1,49 +1,56 @@
...
     attributes:
       value: |
-        Thank you for reporting an issue.
-        If you think we can make Dgraph better, describe your feature request here.
-        Note: Feature requests are judged based on user experience and modeled on 
-        [Go Experience Reports](https://github.com/golang/go/wiki/ExperienceReports). 
-        These reports should focus on the problems - they should not focus on and need not propose solutions.
-  - type: dropdown
+        You have an idea for making Dgraph better? We'd love to hear about it.
+        Please search the issue list first as there is a chance that someone else had the same idea.
```
Comment (skrdgraph) [inline .github/ISSUE_TEMPLATE/feature_request.yml:10]: typo `someone else had the same idea.` -> `someone else has had the same idea.`

**Review by skrdgraph — COMMENTED** [review]
small typo fixes

File: .github/ISSUE_TEMPLATE/enhancement.yml (hunk 4 above)
Comment (rderbier) [reply .github/ISSUE_TEMPLATE/enhancement.yml]: done

**Pull request merged**